spotseeker_server/migrations/0023_auto__del_field_item_category__del_field_item_subcategory.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
```

Comment:
The DB fix should be in a different pull request, as it doesn't look like it's part of the caching system.